test/app-luatest/gh_7000_compat_module_test.lua

```diff
@@ -0,0 +1,262 @@
+local t = require('luatest')
+local g = t.group()
+local compat = require('tarantool').compat
+
+local NEW = 'new'
+local OLD = 'old'
+local DEFAULT = 'default'
+
+local reset = function(compat)
+    local f = loadstring(compat.dump('default'))
+    f()
+end
+
+g.before_all(function() reset(compat) end)
+g.after_all( function() reset(compat) end)
+
+local option_1_called = false
+local option_2_called = false
+
+local hot_reload_option_called
+
+local policies = {
```

Comment:
Aren't we agreed to use one term: either option or policy?

Reply:
You said, you didn't care about the tests. Anyway, renamed to `option_def`..

Reply:
Ouch. Maybe I said something like this in a similar context. I usually worry about tests less then about the user API/the code, but, of course, I glance over them.